go/vt/vtgate/vtgate.go

```diff
@@ -804,6 +806,45 @@ func (vtg *VTGate) GetSrvKeyspace(ctx context.Context, keyspace string) (*topoda
 	return vtg.resolver.toposerv.GetSrvKeyspace(ctx, vtg.resolver.cell, keyspace)
 }
 
+// MessageStream is part of the vtgate service API.
```

Comment:
Maybe add a comment that this is sent to the resolver, so we can stream from the right shards.

And on the next one, MessageAck, say it's sent to the v3 router directly.